examples/langmuir_turbulence.jl

```diff
@@ -176,16 +171,14 @@ set!(model, u=uᵢ, w=wᵢ, b=bᵢ)
 # We use the `TimeStepWizard` for adaptive time-stepping
 # with a Courant-Freidrichs-Lewy (CFL) number of 1.0,
 
-using Oceananigans.Utils
-
 wizard = TimeStepWizard(cfl=1.0, Δt=45.0, max_change=1.1, max_Δt=1minute)
 
 # ### Nice progress messaging
 #
 # We define a function that prints a helpful message with
 # maximum absolute value of ``u, v, w`` and the current wall clock time.
 
-using Oceananigans.Diagnostics, Printf
+using Printf
```

Comment:
Again, do we need `Printf` in "Install dependencies"? Or does it come with basic Julia installation...? I'm not sure...